app/adapters/signal_adapter/inbound.rb

```diff
@@ -68,7 +68,12 @@ def initialize_message(signal_message)
 
       data_message = signal_message.dig(:envelope, :dataMessage)
 
-      message = Message.new(text: data_message[:message], sender: sender)
+      reaction = data_message[:reaction]
+      return nil if reaction && reaction[:isRemove]
+
+      message_text = reaction ? reaction[:emoji] : data_message[:message]
```

Comment:
What does rubocop say about ternary operators in ruby? Isn't it used to be written with an `if` oneliner?

Reply:
Rubocop didn't complain and I just looked it up - it only has rules for [Multiline ternary operators ](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MultilineTernaryOperator) in which case it requests the `if` `then` `else` constructs, but it explicitly marks `a = cond ? b : c` as good. (And in my opinion that's also more common and readable for a ternary operator with little complexity.)